docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang
```

Comment:
what version?

Reply:
if we move gobgp installation to infra docker, it will require port 179, not sure if this affects in somehow to actual or futures requirements for infra docker. Otherwise I don't see any cons.

Reply:
I am not saying that we should move gobgp to cn-infra docker.
We should probably talk about this using IM or webex.